### PR TITLE
ohci:add usb ohci for power

### PR DESCRIPTION
**Pull request by MiriamDeng**
Add usb ohci  for power

ID:1888208

Signed-off-by: MiriramDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
@YongxueHong could you help to review it ? Thanks

File: virttest/qemu_devices/qcontainer.py [hunk 1]
```diff
@@ -1406,7 +1406,10 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus,
                              ' for usb disk %s', name)
                 bus = None
             if fmt == 'usb1':
-                dev_parent = {'type': 'uhci'}
+                if arch.ARCH in ('ppc64', 'ppc64le'):
+                    dev_parent = {'type': 'ohci'}
+                else:
+                    dev_parent = {'type': 'uhci'}
```
Comment (YongxueHong) [inline virttest/qemu_devices/qcontainer.py:1412]:
```suggestion
                dev_parent = {'type': 'uhci'}
                if arch.ARCH in ('ppc64', 'ppc64le'):
                    dev_parent = {'type': 'ohci'}
```

**Comment by MiriamDeng** [comment]
Update it and could you please review it again ? Thanks a lot. @YongxueHong

**Comment by YongxueHong** [comment]
A small issue for commit header:
`ohci:add usb ohci for power `
Better to `qcontainer: Add usb ohci for power `

**Comment by MiriamDeng** [comment]
@YongxueHong , could you help to review it again ? Thanks a lot.

**Review by YongxueHong — APPROVED** [review]

**Comment by YongxueHong** [comment]
Hi @PaulYuuu 
Please help to review, thanks.

**Comment by PaulYuuu** [comment]
LGTM, it's the correct USB controller on PPC.

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**